### PR TITLE
Ccache

### PR DESCRIPTION
**Pull request by jorickert**

### CODE REVIEW

File: rsync-homedir-excludes.txt [hunk 1]
```diff
@@ -280,3 +280,22 @@ nohup.out
...
+.ccache/7
+.ccache/8
+.ccache/9
+.ccache/a
+.ccache/b
+.ccache/c
+.ccache/d
+.ccache/e
+.ccache/f
+.ccache/tmp
```
Comment (rubo77) [inline rsync-homedir-excludes.txt]: Can't you use a wildcard here?

Which would be the config file you want to keep?

Comment (jorickert) [reply rsync-homedir-excludes.txt]: The file i want to keep is .ccache/ccache.conf

Comment (rubo77) [reply rsync-homedir-excludes.txt]: Can you test, if a question mark works for the one-letter folders?

Comment (jorickert) [reply rsync-homedir-excludes.txt]: A question mark does exactly what i want it to do.

**Pull request merged**